github_connector_odoo/models/odoo_module_version.py

```diff
@@ -391,3 +391,25 @@ def create_or_update_from_manifest(
             except Exception as e:
                 _logger.error(
                     'Unable to read the OCA icon image, error is %s' % e)
+
+    @api.model
+    def cron_clean_odoo_module_version(self):
+        module_versions = self.search([])
+        module_versions.clean_odoo_module_version()
+
+    @api.multi
+    def clean_odoo_module_version(self):
+        for module_version in self:
+            module_ver_path = os.path.join(
+                module_version.repository_branch_id.local_path,
+                module_version.technical_name)
+            if os.path.exists(module_ver_path):
+                continue
+            module_version._process_clean_module_version()
```

Comment:
why not just move_version.unlink() ? and removing the function ``_process_clean_module_version``.